### PR TITLE
Add a syndicate stamp

### PR DESCRIPTION
**Pull request by Veritius**
## About the PR 
Adds a syndicate stamp that can be bought from the uplink. Useful for making sure Command knows that the ransom letter is not a prank.

**Changelog**

:cl:
- add: The Syndicate has distributed a special stamp to their operatives, purchasable through the Uplink.

### CODE REVIEW

**Comment by Fishfish458** [comment]
I can try to throw together something for a custom stamp, a red S or something

**Comment by moonheart08** [comment]
Should have a unique icon.

**Comment by Veritius** [comment]
stamp probably needs a unique texture to differentiate it from the nanotrasen-issued ones but I am not a good artist so
when you draw it

**Review by mirrorcult — APPROVED** [review]

File: Resources/Textures/Objects/Misc/bureaucracy.rsi/note.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+TODO: Someone needs to split this up at some point. This RSI is a mess.
```
Comment (mirrorcult) [inline Resources/Textures/Objects/Misc/bureaucracy.rsi/note.txt:1]: just make a github issue dont add random txt files

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**